docs/configuration.md

```diff
@@ -714,6 +714,15 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.deploy.maxCoresPerExecutor</code></td>
+  <td>(infinite)</td>
+  <td>
+    The maximum number of cores given to the executor. When this parameter is set, Spark will try to 
+    run more than 1 executors on each worker in standalone mode; otherwise, only one executor is 
+    launched on each worker.
```

Comment:
We should note that this is 1 executor per application. Technically a worker can still run multiple executors if they belong to different applications. I would rephrase this as:

"""
The maximum number of cores given to an executor. When this parameter is set, multiple executors from the same application may run on the same worker, each with cores equal to or fewer than the configured value. Otherwise, at most one executor per application may run on each worker, as that executor will acquire all the worker's cores by default.

This is used in standalone mode only.
"""